elastic/kibana
# [RAC] Persistent timeline fields fix

## Summary

Solves the issue: https://github.com/elastic/kibana/issues/110524

- Fixes the local storage timeline, it had a circular dependency inside the `filterManager` field, causing an error when trying to `JSON.stringify` in the local storage util module. The field has been cleaned, together with some other fields that we do not need to store.

To test:

- Open a new private window and go to `/app/security` and then:

1. Open Alerts table on the Detections.
2. Change any field using the field modal.
3. Check the `timeline` key is saved in the localStorage (Application/LocalStorage section in chrome dev tools).
4. Refresh the page and check the field change has persisted.

Repeat the same 1-5 steps for:
- Alerts table on the Rule Details page
- Events and External alerts tables on the Host Details page
- Events and External alerts tables on the Host Page
- External alerts on the Network page
